Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.5
```

Comment:
why python3.5? stale ubuntu package repo listings?

Reply:
That's what they want, heh - https://github.com/ProtonVPN/linux-cli/blob/master/CONTRIBUTING.md

Reply:
> 2. Ensure that your code is compatible with the lowest Python Version supported by this project (Currently 3.5)

ha - so it shall be